misk/web/tabs/config/src/components/ConfigComponent.tsx

```diff
@@ -10,9 +10,11 @@ export interface IConfigResource {
 }
 
 export interface IConfigProps {
-  resources: string
+  resources?: string
 }
 
+const cssH3 = css({ fontFamily: "Fira Code, Menlo" })
```

Comment:
can we pull changes like this out to a standard/main stylesheet unless there's a specific reason to only have it here

Reply:
If we want it specifically here, perhaps we can make use of `styled-components` instead?

Reply:
That's something I'll consider in a follow up. I don't think we gain/lose much by having occasional one-off css-in-JS styling. Notably, this syntax is still css-in-js using `@emotion/core` instead of `styled-components` because of better performance and testing story.

Reply:
Ahh awesome! I'll read up more about `@emotion/core` since it sounds like a better option :)